### PR TITLE
I will find my way onto the page

### PR DESCRIPTION
**Pull request by zefwang**

### CODE REVIEW

File: src/content/team/team.json [hunk 1]
```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Daniel Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Boolin"
```
Comment (justineluo) [inline src/content/team/team.json:80]: not a real role

**Review by justineluo — CHANGES_REQUESTED** [review]

**Review by dfarooq610 — APPROVED** [review]

File: src/content/team/team.json [hunk 2]
```diff
@@ -73,6 +73,19 @@
...
+      },
+      "profileImage": "./profileImages/daniel.png",
+      "socialMedia": {
+        "email": "wang.zef@northeastern.edu",
+        "linkedIn": "https://www.linkedin.com/in/zefeng-daniel-wang/",
+        "portfolio": "https://zefeng-wang.com"
```
Comment (jia614) [inline src/content/team/team.json:86]: broken link

File: src/content/team/team.json (hunk 1 above)
Comment (jia614) [reply src/content/team/team.json:80]: +1

**Review by jia614 — CHANGES_REQUESTED** [review]
does not lgtm

File: src/content/team/team.json (hunk 1 above)
Comment (jia614) [reply src/content/team/team.json:80]: +10000

File: src/content/team/team.json [hunk 3]
```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Daniel Wang",
```
Comment (justineluo) [inline src/content/team/team.json:77]: not a real person

Comment (jia614) [reply src/content/team/team.json:77]: +1

File: src/content/team/team.json [hunk 4]
```diff
@@ -73,6 +73,19 @@
...
+      "team": {
+        "name": "Eboard",
```
Comment (justineluo) [inline src/content/team/team.json:79]: No longer accepting applications

File: src/content/team/team.json [hunk 5]
```diff
@@ -73,6 +73,19 @@
...
+        "role": "Head of Boolin"
+      },
+      "profileImage": "./profileImages/daniel.png",
```
Comment (jia614) [inline src/content/team/team.json:82]: picture should be retaken, don't know who this is

**Review by jia614 — CHANGES_REQUESTED** [review]

File: src/content/team/team.json [hunk 6]
```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Zefeng (Daniel) Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Boolin | Dolphin of the East"
```
Comment (emaela) [inline src/content/team/team.json:80]: Screen Shot 2022-04-24 at 12 30 11 AM

Comment (jia614) [reply src/content/team/team.json:80]: so true king

Comment (justineluo) [reply src/content/team/team.json:80]: Please check your spelling and rerequest

File: src/content/team/team.json (hunk 1 above)
Comment (stefan-philip) [reply src/content/team/team.json]: Bro called the branch "patch-1"😭

File: src/content/team/team.json [hunk 7]
```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Zefeng (Daniel) Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Booling | Dolphin of the East"
```
Comment (justineluo) [inline src/content/team/team.json:80]: Screen Shot 2022-04-24 at 12 33 46 AM

**Comment by zefwang** [comment]
For all the haters out there - https://www.dictionary.com/e/slang/boolin/#:~:text=Boolin'%20means%20%E2%80%9Changing%20out%E2%80%9D,It%20comes%20from%20gang%20culture.

File: src/content/team/team.json (hunk 7 above)
Comment (zefwang) [reply src/content/team/team.json:80]: pls see above comment

Comment (jia614) [inline src/content/team/team.json:80]: what is `Booling`? dictionary does not recognize it as a word

Comment (justineluo) [reply src/content/team/team.json:80]: Did you mean boiling? booing?

**Comment by VKong6019** [comment]
deez noots

File: src/content/team/team.json (hunk 7 above)
Comment (jia614) [reply src/content/team/team.json:80]: book-keeping?

Comment (zefwang) [reply src/content/team/team.json:80]: https://www.dictionary.com/e/slang/boolin/#:~:text=Boolin'%20means%20%E2%80%9Changing%20out%E2%80%9D,It%20comes%20from%20gang%20culture.

Comment (jia614) [reply src/content/team/team.json:80]: big-bootying?

Comment (justineluo) [reply src/content/team/team.json:80]: bowling?

**Review by dfarooq610 — CHANGES_REQUESTED** [review]
Consider being better

File: src/content/team/team.json (hunk 1 above)
Comment (zefwang) [reply src/content/team/team.json]: bro blame github - they autonamed the branch for me lmao

File: src/content/team/team.json (hunk 7 above)
Comment (jia614) [reply src/content/team/team.json]: that's not the right dictionary

**Review by dfarooq610 — COMMENTED** [review]
Ty for the changes: LGTM (Large Giraffes Terrorize Me)

**Review by dfarooq610 — APPROVED** [review]

**Comment by zefwang** [comment]
If you're an applicant looking through our PRs, Sandbox loves to #be-kind and #learn-together :)

**Pull request closed without merging**